tools/ui-components/src/button.test.js

```diff
@@ -0,0 +1,29 @@
+/* global expect */
+/* global jest */
```

Comment:
I think I will raise a PR to clean these up for the entire codebase (with [eslint-plugin-jest](https://www.npmjs.com/package/eslint-plugin-jest)), unless anyone has already worked on this.